liquidctl/cli.py

```diff
@@ -45,6 +46,10 @@
   --version                      Display the version number
   --help                         Show this message
 
+Config file options:
+  -f, --file <file>              Select a specific config file to use
+  --ignore-config                Do not use any of the config files
```

Comment:
`--ignore-config` is probably unnecessary, `-f /dev/null` or `-f <some empty file>` are already sufficient for this very niche use case.